mappings/net/minecraft/data/family/BlockFamily.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_5794 net/minecraft/data/family/BlockFamily
 	FIELD field_28529 generateRecipes Z
 	FIELD field_28530 group Ljava/lang/String;
 	FIELD field_28531 unlockCriterionName Ljava/lang/String;
+	FIELD field_40591 featureSet Lnet/minecraft/class_7699;
```

Comment:
Is this required or enabled features btw? Should be clarified.

Reply:
```suggestion
	FIELD field_40591 requiredFeatures Lnet/minecraft/class_7699;
```